org.thingpedia.iot.thermostat/dataset.tt

```diff
@@ -0,0 +1,130 @@
+dataset @org.thingpedia.iot.thermostat language "en" {
+
+    program := now => @org.thingpedia.iot.thermostat.humidity() => notify
+    #_[utterances=["is it humid?",
+                   "check if it is humid",
+                   "what does my humidity sensor say?",
+                   "how humid is it?",
+                   "what is the level of humidity?",
+                   "how high is the humidity?"]];
+
+    stream := monitor @org.thingpedia.iot.thermostat.humidity()
+    #_[utterances=["when the humidity changes"]];
```

Comment:
We already have a humidity sensor, so we don't need this.

Reply:
Fixed in latest commit